README.md

```diff
@@ -233,6 +233,99 @@ end
 <%== pagy_nav(@pagy) %>
 ```
 
+### Customization
+
+A common pattern we use is to have a cell at the end of the table for actions. For example:
+
+```html
+<table class="action-table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th class="actions"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Alice</td>
+      <td class="actions">
+        <a href="/people/1/edit">Edit</a>
+        <a href="/people/1" method="delete">Delete</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+```
+
+You can write a custom builder that helps generate this type of table by adding the required classes and adding helpers
+for generating the actions. This allows for a declarative table syntax, something like this:
+
+```erb
+<%= table_with(collection: collection, builder: Test::ActionTable) do |row| %>
+  <%= row.cell :name %>
+  <%= row.actions do |cell| %>
+    <%= cell.action "Edit", :edit %>
+    <%= cell.action "Delete", :delete, method: :delete %>
+  <% end %>
+<% end %>
+```
+
+And the custom builder:
+
+```ruby
+class ActionTable < Katalyst::Tables::Frontend::TableBuilder
+  def build(&block)
+    (@html_options[:class] ||= []) << "action-table"
+    super
+  end
+
+  def table_header_row(builder = ActionHeaderRow, &block)
+    super
+  end
+
+  def table_header_cell(method, builder = ActionHeaderCell, **options)
+    super
+  end
+
+  def table_body_row(object, builder = ActionBodyRow, &block)
+    super
+  end
+
+  def table_body_cell(object, method, builder = ActionBodyCell, **options, &block)
+    super
+  end
+
+  class ActionHeaderRow < Katalyst::Tables::Frontend::Builder::HeaderRow
+    def actions(&block)
+      cell(:actions, class: "actions", label: "")
+    end
+  end
+
+  class ActionHeaderCell < Katalyst::Tables::Frontend::Builder::HeaderCell
```

Comment:
**Question:** I know for empty methods the approach is to bring it all onto a single line; however is the same called for empty classes in this given scenario?